README.md

```diff
@@ -182,6 +182,45 @@ feed:
         - updates
 ```
 
+## Tags
+
+To automatically generate feeds for each tag you apply to your posts you can add a tags setting to your config:
+
+```yml
+feed:
+  tags:
+```
```

Comment:
This is incorrect. This does not mean `{"feed"=>{"tags" => {}}}`
This evaluates to `{"feed"=>{"tags" => nil}}`

Instead, recommend enabling via a boolean `true`:
```yml
feed:
  tags: true
```